2023-04-27 15:56:35,740 INFO org.mongodb.driver.cluster [cluster-ClusterId{value='644a3c6a2f3ef0650dc1dc38', description='null'}-ac-vs3fwnk-shard-00-01.xufo0j7.mongodb.net:27017] Exception in monitor thread while connecting to server ac-vs3fwnk-shard-00-01.xufo0j7.mongodb.net:27017
com.mongodb.MongoSocketReadTimeoutException: Timeout while receiving message
	at com.mongodb.connection.netty.NettyStream$InboundBufferHandler.exceptionCaught(NettyStream.java:427)
	at io.netty.channel.AbstractChannelHandlerContext.invokeExceptionCaught(AbstractChannelHandlerContext.java:346)
	at io.netty.channel.AbstractChannelHandlerContext.invokeExceptionCaught(AbstractChannelHandlerContext.java:325)
	at io.netty.channel.AbstractChannelHandlerContext.fireExceptionCaught(AbstractChannelHandlerContext.java:317)
	at com.mongodb.connection.netty.NettyStream$ReadTimeoutTask.run(NettyStream.java:564)
	at io.netty.util.concurrent.PromiseTask.runTask(PromiseTask.java:98)
	at io.netty.util.concurrent.ScheduledFutureTask.run(ScheduledFutureTask.java:153)
	at io.netty.util.concurrent.AbstractEventExecutor.runTask(AbstractEventExecutor.java:174)
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:167)
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:470)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:569)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: io.netty.handler.timeout.ReadTimeoutException: null
2023-04-27 15:56:35,890 INFO org.mongodb.driver.cluster [cluster-ClusterId{value='644a3c6a2f3ef0650dc1dc38', description='null'}-ac-vs3fwnk-shard-00-00.xufo0j7.mongodb.net:27017] Exception in monitor thread while connecting to server ac-vs3fwnk-shard-00-00.xufo0j7.mongodb.net:27017
com.mongodb.MongoSocketReadTimeoutException: Timeout while receiving message
	at com.mongodb.connection.netty.NettyStream$InboundBufferHandler.exceptionCaught(NettyStream.java:427)
	at io.netty.channel.AbstractChannelHandlerContext.invokeExceptionCaught(AbstractChannelHandlerContext.java:346)
	at io.netty.channel.AbstractChannelHandlerContext.invokeExceptionCaught(AbstractChannelHandlerContext.java:325)
	at io.netty.channel.AbstractChannelHandlerContext.fireExceptionCaught(AbstractChannelHandlerContext.java:317)
	at com.mongodb.connection.netty.NettyStream$ReadTimeoutTask.run(NettyStream.java:564)
	at io.netty.util.concurrent.PromiseTask.runTask(PromiseTask.java:98)
	at io.netty.util.concurrent.ScheduledFutureTask.run(ScheduledFutureTask.java:153)
	at io.netty.util.concurrent.AbstractEventExecutor.runTask(AbstractEventExecutor.java:174)
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:167)
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:470)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:569)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: io.netty.handler.timeout.ReadTimeoutException: null
2023-04-27 15:56:36,155 INFO org.mongodb.driver.cluster [cluster-ClusterId{value='644a3c6a2f3ef0650dc1dc38', description='null'}-ac-vs3fwnk-shard-00-02.xufo0j7.mongodb.net:27017] Exception in monitor thread while connecting to server ac-vs3fwnk-shard-00-02.xufo0j7.mongodb.net:27017
com.mongodb.MongoSocketReadTimeoutException: Timeout while receiving message
	at com.mongodb.connection.netty.NettyStream$InboundBufferHandler.exceptionCaught(NettyStream.java:427)
	at io.netty.channel.AbstractChannelHandlerContext.invokeExceptionCaught(AbstractChannelHandlerContext.java:346)
	at io.netty.channel.AbstractChannelHandlerContext.invokeExceptionCaught(AbstractChannelHandlerContext.java:325)
	at io.netty.channel.AbstractChannelHandlerContext.fireExceptionCaught(AbstractChannelHandlerContext.java:317)
	at com.mongodb.connection.netty.NettyStream$ReadTimeoutTask.run(NettyStream.java:564)
	at io.netty.util.concurrent.PromiseTask.runTask(PromiseTask.java:98)
	at io.netty.util.concurrent.ScheduledFutureTask.run(ScheduledFutureTask.java:153)
	at io.netty.util.concurrent.AbstractEventExecutor.runTask(AbstractEventExecutor.java:174)
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:167)
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:470)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:569)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: io.netty.handler.timeout.ReadTimeoutException: null
2023-04-27 15:56:39,913 INFO org.mongodb.driver.cluster [cluster-ClusterId{value='644a3c6a2f3ef0650dc1dc38', description='null'}-ac-vs3fwnk-shard-00-01.xufo0j7.mongodb.net:27017] Exception in monitor thread while connecting to server ac-vs3fwnk-shard-00-01.xufo0j7.mongodb.net:27017
com.mongodb.MongoSocketException: This is usually a temporary error during hostname resolution and means that the local server did not receive a response from an authoritative server (ac-vs3fwnk-shard-00-01.xufo0j7.mongodb.net)
	at com.mongodb.ServerAddress.getSocketAddresses(ServerAddress.java:217)
	at com.mongodb.connection.netty.NettyStream.openAsync(NettyStream.java:163)
	at com.mongodb.connection.netty.NettyStream.open(NettyStream.java:154)
	at com.mongodb.internal.connection.InternalStreamConnection.open(InternalStreamConnection.java:183)
	at com.mongodb.internal.connection.DefaultServerMonitor$ServerMonitorRunnable.lookupServerDescription(DefaultServerMonitor.java:198)
	at com.mongodb.internal.connection.DefaultServerMonitor$ServerMonitorRunnable.run(DefaultServerMonitor.java:158)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.net.UnknownHostException: This is usually a temporary error during hostname resolution and means that the local server did not receive a response from an authoritative server (ac-vs3fwnk-shard-00-01.xufo0j7.mongodb.net)
	at java.base/java.net.Inet6AddressImpl.lookupAllHostAddr(Native Method)
	at java.base/java.net.Inet6AddressImpl.lookupAllHostAddr(Inet6AddressImpl.java:52)
	at java.base/java.net.InetAddress$PlatformResolver.lookupByName(InetAddress.java:1048)
	at java.base/java.net.InetAddress.getAddressesFromNameService(InetAddress.java:1648)
	at java.base/java.net.InetAddress$NameServiceAddresses.get(InetAddress.java:997)
	at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1638)
	at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1504)
	at com.mongodb.ServerAddress.getSocketAddresses(ServerAddress.java:209)
	... 6 common frames omitted
2023-04-27 15:56:40,020 INFO org.mongodb.driver.cluster [cluster-ClusterId{value='644a3c6a2f3ef0650dc1dc38', description='null'}-ac-vs3fwnk-shard-00-00.xufo0j7.mongodb.net:27017] Exception in monitor thread while connecting to server ac-vs3fwnk-shard-00-00.xufo0j7.mongodb.net:27017
com.mongodb.MongoSocketException: This is usually a temporary error during hostname resolution and means that the local server did not receive a response from an authoritative server (ac-vs3fwnk-shard-00-00.xufo0j7.mongodb.net)
	at com.mongodb.ServerAddress.getSocketAddresses(ServerAddress.java:217)
	at com.mongodb.connection.netty.NettyStream.openAsync(NettyStream.java:163)
	at com.mongodb.connection.netty.NettyStream.open(NettyStream.java:154)
	at com.mongodb.internal.connection.InternalStreamConnection.open(InternalStreamConnection.java:183)
	at com.mongodb.internal.connection.DefaultServerMonitor$ServerMonitorRunnable.lookupServerDescription(DefaultServerMonitor.java:198)
	at com.mongodb.internal.connection.DefaultServerMonitor$ServerMonitorRunnable.run(DefaultServerMonitor.java:158)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.net.UnknownHostException: This is usually a temporary error during hostname resolution and means that the local server did not receive a response from an authoritative server (ac-vs3fwnk-shard-00-00.xufo0j7.mongodb.net)
	at java.base/java.net.Inet6AddressImpl.lookupAllHostAddr(Native Method)
	at java.base/java.net.Inet6AddressImpl.lookupAllHostAddr(Inet6AddressImpl.java:52)
	at java.base/java.net.InetAddress$PlatformResolver.lookupByName(InetAddress.java:1048)
	at java.base/java.net.InetAddress.getAddressesFromNameService(InetAddress.java:1648)
	at java.base/java.net.InetAddress$NameServiceAddresses.get(InetAddress.java:997)
	at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1638)
	at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1504)
	at com.mongodb.ServerAddress.getSocketAddresses(ServerAddress.java:209)
	... 6 common frames omitted
2023-04-27 15:56:40,283 INFO org.mongodb.driver.cluster [cluster-ClusterId{value='644a3c6a2f3ef0650dc1dc38', description='null'}-ac-vs3fwnk-shard-00-02.xufo0j7.mongodb.net:27017] Exception in monitor thread while connecting to server ac-vs3fwnk-shard-00-02.xufo0j7.mongodb.net:27017
com.mongodb.MongoSocketException: This is usually a temporary error during hostname resolution and means that the local server did not receive a response from an authoritative server (ac-vs3fwnk-shard-00-02.xufo0j7.mongodb.net)
	at com.mongodb.ServerAddress.getSocketAddresses(ServerAddress.java:217)
	at com.mongodb.connection.netty.NettyStream.openAsync(NettyStream.java:163)
	at com.mongodb.connection.netty.NettyStream.open(NettyStream.java:154)
	at com.mongodb.internal.connection.InternalStreamConnection.open(InternalStreamConnection.java:183)
	at com.mongodb.internal.connection.DefaultServerMonitor$ServerMonitorRunnable.lookupServerDescription(DefaultServerMonitor.java:198)
	at com.mongodb.internal.connection.DefaultServerMonitor$ServerMonitorRunnable.run(DefaultServerMonitor.java:158)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.net.UnknownHostException: This is usually a temporary error during hostname resolution and means that the local server did not receive a response from an authoritative server (ac-vs3fwnk-shard-00-02.xufo0j7.mongodb.net)
	at java.base/java.net.Inet6AddressImpl.lookupAllHostAddr(Native Method)
	at java.base/java.net.Inet6AddressImpl.lookupAllHostAddr(Inet6AddressImpl.java:52)
	at java.base/java.net.InetAddress$PlatformResolver.lookupByName(InetAddress.java:1048)
	at java.base/java.net.InetAddress.getAddressesFromNameService(InetAddress.java:1648)
	at java.base/java.net.InetAddress$NameServiceAddresses.get(InetAddress.java:997)
	at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1638)
	at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1504)
	at com.mongodb.ServerAddress.getSocketAddresses(ServerAddress.java:209)
	... 6 common frames omitted
